apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -327,70 +344,99 @@ Func CameraPipe::build() {
     Func corrected = color_correct(demosaiced->output);
     Func processed = apply_curve(corrected);
 
+    Pipeline p(processed);
+
     // Schedule
-    Expr out_width = processed.output_buffer().width();
-    Expr out_height = processed.output_buffer().height();
-    // In HVX 128, we need 2 threads to saturate HVX with work,
-    //and in HVX 64 we need 4 threads, and on other devices,
-    // we might need many threads.
-    Expr strip_size;
-    if (get_target().has_feature(Target::HVX_128)) {
-        strip_size = processed.output_buffer().dim(1).extent() / 2;
-    } else if (get_target().has_feature(Target::HVX_64)) {
-        strip_size = processed.output_buffer().dim(1).extent() / 4;
-    } else {
-        strip_size = 32;
-    }
-    strip_size = (strip_size / 2) * 2;
+    if (!auto_schedule) {
```

Comment:
Instead of 
```
if (!auto_schedule) {
} else {
}
```
it's probably better to do this:
```
if (auto_schedule) {
} else {
}
```
to match the other if-else auto-scheduler pattern in apps.